subprojects/docs/src/docs/release/notes.md

```diff
@@ -40,6 +40,38 @@ details of 2
 ## n
 -->
 
+## Show location of Java crash log
+
+Sometimes, the Gradle daemon crashes due to some bugs in Java, Gradle or some third party plugin.
+When Java crashes, it writes a [fatal error log](https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/felog.html) to the current working directory.
```

Comment:
```suggestion
When the Java VM crashes, a [fatal error log](https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/felog.html) is created in the working directory of the daemon process.
```

Reply:
Should we link the Java 8 docs here, or something newer perhaps?

Reply:
I am linking to the Java 13 docs now.

I also applied your suggestion manually.

Reply:
I guess that's fine, though maybe we should link to LTE versions only? Probably nitpicking.

Reply:
Java 11 it is.